CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+1.5.0
```

Comment:
jbuck:master was already 1.5.0.

Reply:
Master is at 1.4.0 at the moment

Reply:
https://github.com/jbuck/assume-aws-role/blob/master/package.json#L3

Reply:
oh, but in npm, 1.5.0 is already released.